packages/browser/src/tracing/browserTracingIntegration.ts

```diff
@@ -206,6 +216,11 @@ export const browserTracingIntegration = ((_options: Partial<BrowserTracingOptio
   if (enableLongTask) {
     startTrackingLongTasks();
   }
+
+  if (enableLongAnimationFrame) {
+    startTrackingLongAnimationFrames();
```

Comment:
IMO if we are tracking long animation frames, we don't need to track long tasks (they should overlap). What do you think?

Reply:
Agreed. Will add a guard for this